src/App.js

```diff
@@ -1,49 +1,48 @@
 import './App.css';
 import axios from 'axios';
 import React, { useEffect, useState } from 'react';
-import Moment from 'moment';
+import NotificationsFlyout, { anchorPoints }  from "@hig/notifications-flyout";
+import Timestamp from "@hig/timestamp";
 
 function App() {
-
+  let notificationData = [];
   const [APIData, setAPIData] = useState([]);
   useEffect(() => {
     axios.get(`http://demo9540080.mockable.io/notifications`)
       .then((response) => {
         setAPIData(response.data.notifications);
       })
   }, []);
-
+  
+  for (let i = 0; i < APIData.length; i++) 
+  {
+	var notificationItem = {
+							  id: APIData[i].id,
+							  featured: true,
```

Comment:
What does featured mean?

Reply:
A featured notification can be dismissed.